network.py

```diff
@@ -23,37 +33,90 @@ def one_hot(y, output_size):
     return t
 
 
-if __name__ == "__main__":
+def check_loss(network, x_check_cost, y_check_cost):
```

Comment:
Ja bym tutaj nie umieszczał `_check_lost` w nazwie. To nic nie wnosi do rozumienia tego, co się dzieje w funkcji.

Reply:
dobra może być x i y